experimental/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ All notable changes to experimental packages in this project will be documented
 
 ### :boom: Breaking Change
 
+* Rename @opentelemetry/sdk-metrics-base package to @opentelemetry/sdk-metrics  [#3162](https://github.com/open-telemetry/opentelemetry-js/pull/3162) @hectorhdzg
```

Comment:
is that really a breaking change ? I mean the previous package is still there and it works no ?

Reply:
Packages need to update their code reference to the renamed package when upgrading to the new version, same as upgrading to a version with breaking change. I'd find it is ok to count it as a breaking change, or notable change.